stdlib/socketserver.pyi

```diff
@@ -39,50 +41,25 @@ class BaseServer:
     def close_request(self, request: _RequestType) -> None: ...  # undocumented
 
 class TCPServer(BaseServer):
+    allow_reuse_port: bool
+    request_queue_size: int
```

Comment:
- Where is `allow_reuse_port` defined? I can't find it in Python 3.9 socketserver.py.
- Shouldn't these be `ClassVar`s?

Reply:
I also thought that they should be `ClassVar`s at first, but [the docstring](https://github.com/python/cpython/blob/8fb36494501aad5b0c1d34311c9743c60bb9926c/Lib/socketserver.py#L183) specifically states that it's okay for instances to override these attributes as well as subclasses. So I think @sobolevn's current annotation of them as instance attributes is probably better.